tests/test_linter_endpoint.py

```
@@ -0,0 +1,110 @@
+'''
+
+
+def test_linter_endpoint(url="http://localhost:4000/api"):
+    """Test the linter endpoint with various contracts"""
+
+    print("Testing GenVM Linter Endpoint...")
+    print("-" * 50)
+
+    # Test 1: Contract with issues
+    print("\n1. Testing contract with issues:")
+    response = requests.post(url, json={
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)